TimeCalc.java

```diff
@@ -1,5 +1,31 @@
 public class TimeCalc {
     public static void main(String[] args) {
         // Replace this comment with your code
+        String time="";
+        int minutesToAdd=0,hh=0,mm=0;
+        int totalHours=0, totalMinutes=0;

```

Comment:
no need to declare them at the beginning, you can declare them when using them :)